src/clients/api/mutations/useEnterMarkets.ts

```diff
@@ -0,0 +1,29 @@
+import { useMutation, MutationObserverOptions } from 'react-query';
+
+import { enterMarkets, IEnterMarketsInput, EnterMarketsOutput } from 'clients/api';
+import FunctionKey from 'constants/functionKey';
+import { useComptroller } from 'hooks/useContract';
+
+const useEnterMarkets = (
+  // TODO: use custom error type
```

Comment:
do we have tickets for this TODOs? maybe we can have links in comments to proper tasks?

Reply:
We don't, that's a good point.
I've created it and added the reference inside the relevant comments 👍